src/app/components/Citations/APACitation.jsx

```diff
@@ -0,0 +1,186 @@
+      }
+
+      return `${lastName}, ${APACitation.getInitials(firstNames)}`;
+    }).join(', ');
+  }
+
+  /**
+   * Return a government report citation when govReportStatus is true
+   * Citations for these documents are somewhat simpler than for monographs
+   *
+   * Standard format
+   * Authors OR Editors (Ed.) (Publication Year) <em>Title: Subtitle</em> Publishers. Link
+   *
+   * @returns {object} Element to be displayed
+   */
+  returnGovernmentReport() {
+    let responsibilityStatement = '';
+    if (this.props.agents.authors.length > 0) {
+      responsibilityStatement = this.authors;
+    } else if (this.props.agents.editors.length > 0) {
+      responsibilityStatement = this.editors;
+    }
+
+    return (
+      <>
+        {`${responsibilityStatement}${this.pubYear}. `}
+        <em>{`${this.title} `}</em>
+        {`${this.publishers}. `}
+        {this.link}
+      </>
+    );
+  }
+
+  /**
+   * Returns a formatted citation for a monograph, including the volume if present
+   * Monograph citations can include all metadata passed into this component. Importantly
+   * the editors block can change position depending on the presence of authors
+   *
+   * Authors Format
+   * Authors (Publication Year) <em>Title: SubTitle</em> (Volume) (Illustrators OR Translators) (Edition) (Editors) Publishers. Link
+   * No Authors Format
+   * Editors (Ed.) (Publication Year) <em>Title: SubTitle</em> (Volume) (Illustrators OR Translators) (Edition) Publishers. Link
+   *
+   * @param {boolean} addVolume Set if volume data is present to return in citation
+   *
+   * @returns {object} Formatted element to be displayed
+   */
+  returnMonographCitation(addVolume = false) {
+    let responsibilityStatement = '';
+    if (this.props.agents.authors.length > 0) {
+      responsibilityStatement = this.authors;
+      if (this.props.agents.editors.length > 0) {
+        this.edition = `${this.edition} (${this.editors}).`;
+      }
+    } else if (this.props.agents.editors.length > 0) {
+      responsibilityStatement = this.editors;
+    }
+
+    if (this.props.agents.illustrators.length > 0 && this.props.agents.translators.length > 0) {
+      this.title = `${this.title} (${this.illustrators}, ${this.translators})`;
+    } else if (this.props.agents.illustrators.length > 0) {
+      this.title = `${this.title} (${this.illustators})`;
+    } else if (this.props.agents.translators.length > 0) {
+      this.title = `${this.title} (${this.translators})`;
+    }
+
+    return (
+      <>
+        {`${responsibilityStatement}${this.pubYear}. `}
+        <em>{`${this.title}`}</em>
+        {`${addVolume ? this.volume : ''}`}
+        {`${this.edition} `}
+        {`${this.publishers}. `}
+        {this.link}
+      </>
+    );
+  }
+
+  // Returns appropriate APA citation version
+  render() {
+    const volumeData = this.volume !== '';
+    return (
+      <div className="apa-citation body-small">
```

Comment:
can you add a little margin-bottom to this so that it's not stuck to the title of the next book?

Reply:
What's the best way to do this? I meant to ask because all the CSS lives in design system, right?